addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -1920,6 +1920,10 @@ var FieldOne2Many = FieldX2Many.extend({
             this.recordParams,
             { additionalContext: params.context }
         ));
+
+        if(this.attrs.options && this.attrs.options.no_open)
+            return;
```

Comment:
```suggestion
        if(this.attrs.options && this.attrs.options.no_open) {
            return;
        }
```
Also, there should be a test, and the option should be documented